x/pki/types/key_noc_root_certificates_by_vid_and_skid.go

```diff
@@ -5,12 +5,12 @@ import "encoding/binary"
 var _ binary.ByteOrder
 
```

Comment:
Should we rename the file (get rid of `root`)?

Reply:
Fixed